pipelines/unittests/utils/infoCommit.groovy

```diff
@@ -0,0 +1,36 @@
+String getCommitDate() {
```

Comment:
Estas funciones que ya estaban creadas las agregaste de nuevo por algún motivo?

Reply:
@facumoyanosmf these functions do not exist in this repository, but in the other pipeline repository.